wire-ios/Wire-iOS/Sources/UserInterface/Calling/CallGridView/CallParticipantViews/CallParticipantView.swift

```diff
@@ -162,15 +166,6 @@ final class CallParticipantView: BaseCallParticipantView {
     // MARK: Override Base
 
     override func updateVideoShouldFill(_ shouldFill: Bool) {
-        if shouldFill, videoView == nil {
-            // [WPB-8954] Setup video only when the video really starts to avoid
-            // calls crashing on the iOS 17 simulator.
```

Comment:
I was a bit confused by this condition... `shouldFill` is `true` when the user enables camera, but not screen share. I would expect the condition to check the participant state to see if they have either camera or screen share enabled.

Also... I wasn't able to reproduce the issue described, maybe my testing wasn't rigorous enough. Do you think we should still avoid setting up the video view for simulators? In that case, couldn't we do something like:

```swift
#if targetEnvironment(simulator)
  // don't create video view
#else
  // create video view
#endif
```